tests/scripts/depends.py

```diff
@@ -362,12 +361,12 @@ def test(self, options):
 EXCLUSIVE_GROUPS = {
     'MBEDTLS_SHA512_C': ['-MBEDTLS_SSL_COOKIE_C',
                          '-MBEDTLS_SSL_TLS_C'],
-    'MBEDTLS_ECP_DP_CURVE448_ENABLED': ['-MBEDTLS_ECDSA_C',
-                                        '-MBEDTLS_ECDSA_DETERMINISTIC',
-                                        '-MBEDTLS_ECJPAKE_C',],
-    'MBEDTLS_ECP_DP_CURVE25519_ENABLED': ['-MBEDTLS_ECDSA_C',
-                                          '-MBEDTLS_ECDSA_DETERMINISTIC',
-                                          '-MBEDTLS_ECJPAKE_C'],
+    'PSA_WANT_ECC_MONTGOMERY_448': ['-MBEDTLS_ECDSA_C',
```

Comment:
As mentioned offline now when we run the curves tests  MBEDTLS_ECP_DP_SECP224K1_ENABLED is never disabled. It would be better if it were. We could use EXCLUSIVE_GROUPS to do the job, what do you think?

Reply:
The second option looks good to me with the exclusion of `MBEDTLS_ECP_DP_SECP224K1_ENABLED`. That way the other curves are tested with all code related to SECP224K1 disabled and we do not run tests for SECP224K1. When the configuration option is removed, we will just remove MBEDTLS_ECP_DP_SECP224K1_ENABLED from the list of curve symbols.